src/FileCacheManager.php

```diff
@@ -43,15 +40,14 @@
     /**
      * Create instance.
      *
-     * @param bool   $isEnabled is cache enabled
-     * @param string $cacheFile cache file
-     * @param bool   $linting   is linting enabled
-     * @param array  $rules     array defining rules, format like one for ConfigInterface::setRules
+     * @param CacheHandler $cacheHandler handles read/write operations
+     * @param string       $cacheFile    cache file
```

Comment:
@keradus 

It would probably be cool to get rid of the `$cacheFile` parameter here, but it is used for determining the path to file names relative to it.